src/map.c

```diff
@@ -1113,63 +1113,58 @@ void map_renderSystems( double bx, double by, double x, double y,
  */
 static void map_renderPath( double x, double y, double a )
 {
-   int j;
+   int j, k, sign;
    const glColour *col;
-   GLfloat vertex[8*(2+4)];
-   StarSystem *jsys, *lsys;
+   double w0, w1, x0, y0, x1, y1, h0, h1;
+   GLfloat vertex[(3*2)*(2+4)];
+   StarSystem *sys1, *sys0;
    int jmax, jcur;
 
    if (map_path != NULL) {
-      lsys = cur_system;
+      sys0 = cur_system;
       jmax = pilot_getJumps(player.p); /* Maximum jumps. */
       jcur = jmax; /* Jump range remaining. */
 
-      /* Generate smooth lines. */
-      glLineWidth( CLAMP(1., 4., 2. * map_zoom)*gl_screen.scale );
-
       for (j=0; j<map_npath; j++) {
-         jsys = map_path[j];
+         sys1 = map_path[j];
          if (jcur == jmax && jmax > 0)
             col = &cGreen;
          else if (jcur < 1)
             col = &cRed;
          else
             col = &cYellow;
+         x0 = x + sys0->pos.x * map_zoom;
+         y0 = y + sys0->pos.y * map_zoom;
+         x1 = x + sys1->pos.x * map_zoom;
+         y1 = y + sys1->pos.y * map_zoom;
+         w0 = w1 = map_zoom / hypot( x0-x1, y0-y1 );
+	 w0 *= jcur >= 1 ? 4 : 2;
          jcur--;
+	 w1 *= jcur >= 1 ? 4 : 2;
```

Comment:
whitespace problem?

Reply:
vim strikes again! I am going to engage in some revisionist history, because several commits contain tabs that shouldn't be tabs.